### DISCUSSION TITLE
Changing Metric Plots Properties

### DISCUSSION THREAD

**Discussion by TRDWuisan** [ultralytics/yolov5#11324] [Q&A]
Heya folks, a bit of a basic question here (i hope), but I'm using roboflow's yolov8 tutorial and am wondering how to change the formatting of the metric plots (train/valid losses, f1, precision, recall curves, confusion matrix, mAP curves, etc.). or better yet, how can I call the values/variables that are used to plot these curves as in their tutorial the plots are saved as pngs directly. Thanks :D

**Comment by TRDWuisan** [comment]
mostly solved with the results.csv values, but what about the F1/P/R curves and confusion matrix? Is there a way to find call the values used to plot? Thanks again :D

**Reply by usapeshka** [reply]
Can you splurge some context on how you solved the above please?
